assembl/static2/js/app/components/voteSession/proposalResults.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+
+import { GaugeVoteForProposal, NumberGaugeVoteForProposal } from './gaugeVoteForProposal';
+import TokenVotesResults from './tokenVotesResults';
+import ParticipantsCount from './participantsCount';
+import {
+  findTokenVoteModule,
+  filterGaugeVoteModules,
+  filterNumberGaugeVoteModules,
+  type VoteSpecification
+} from '../../pages/voteSession';
+
+type Props = {
+  description: ?string,
+  id: string,
+  modules: Array<VoteSpecification>,
+  numParticipants: number,
+  title: ?string
+};
+
+class ProposalResults extends React.Component<void, Props, void> {
```

Comment:
Why is a class based component needed here ?

Reply:
Not really needed. I copied the component from proposal.jsx and removed the state. This could be transformed to a functional component indeed.